src/transport/raw/MessageHeader.h

```diff
@@ -49,6 +49,22 @@ enum class EncryptionType
     kAESCCMTagLen16 = 2,
 };
 
+/**
+ *  @brief
+ *    The CHIP Exchange header flag bits.
+ */
+enum class ExFlagValues : uint16_t
+{
+    /// Set when current message is sent by the initiator of an exchange.
+    kExchangeFlag_Initiator = 0x0001,
+
+    /// Set when current message is an acknowledgment for a previously received message.
+    kExchangeFlag_AckId = 0x0002,
```

Comment:
why is this flag part of exchange manager?

Reply:
Yes, this flag is not part of ExchangeManager, I added here since kExchangeFlag_AckId is already defined by spec.

Reply:
In that case we should remove it. I guess, CRMP will use a similar flag. It should go as part of that PR. Same or the next flag.

Reply:
Ack